rudder-server/SOURCES/Makefile

```diff
@@ -162,8 +162,7 @@ install: build rudder-server-version initial-promises initial-ncf
 	mkdir -p $(DESTDIR)/opt/rudder/share/versions/
 	install -m 644 rudder-sources/rudder/webapp/sources/rudder/rudder-core/src/main/resources/reportsSchema.sql $(DESTDIR)/opt/rudder/etc/postgresql/
 	install -m 644 rudder-server.service $(DESTDIR)/usr/lib/systemd/system/
-	install -m 755 rudder-server-root-postinst $(DESTDIR)/opt/rudder/share/package-scripts/
```

Comment:
i would rename to rudder-server-postinst  instead of rudder-postinst since it's linked to rudder-server-postinst

Reply:
indeed